MAUI/Polar-Charts/DataLabels.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: Data labels in .NET MAUI Chart control | Syncfusion
+description: Learn here all about how to configure the data labels and their features in Syncfusion .NET MAUI Chart (SfPolarChart).
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Data Labels in .NET MAUI Chart
+
+Data labels are used to display values related to a chart segment. Values from a data point(x, y) or other custom properties from a data source can be displayed. 
+
+Each data label can be represented by the following:
+
+* Label - displays the segment label content at the (X, Y) point.
+
+## Enable Data Label 
+
+The [ShowDataLabels]() property of a series is used to enable the data labels.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    . . .
+    <chart:PolarLineSeries ItemsSource="{Binding PlantDetails}" 
+                        XBindingPath="Direction"
+                        YBindingPath="Tree" ShowDataLabels="True">
+        </chart:PolarLineSeries>
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+PolarLineSeries series = new PolarLineSeries()
+{
+    ItemsSource = viewModel.PlantDetails,
+    XBindingPath = "Direction",
+    YBindingPath = "Tree",
+    ShowDataLabels = true
+};
+
+chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Data label in MAUI chart](DataLabel_images/MAUI_polar_line_datalabel.png)
+
+Data labels can be customized by using the [DataLabelSettings]() property of chart series. To customize them, you need to create an instance of [PolarDataLabelSettings]() and set it to the [DataLabelSettings]() property. The following properties available in [PolarDataLabelSettings]() can be used to customize the data labels.
```

Comment:
Data labels can be customized using the [DataLabelSettings]() property of chart series. To customize them, you need to create an instance of [PolarDataLabelSettings]() and set it to the [DataLabelSettings]() property. The following properties available in [PolarDataLabelSettings]() can be used to customize the data labels.